pages/overview/roadmap/index.page.tsx

```diff
@@ -0,0 +1,128 @@
+import React from 'react';
+import { DocsHelp } from '~/components/DocsHelp';
+import { getLayout } from '~/components/Sidebar';
+import Head from 'next/head';
+import { Headline1 } from '~/components/Headlines';
+import { SectionContext } from '~/context';
+import roadmap from '~/data/roadmap.json';
+
+const statusColors = {
+  'In Progress': 'bg-green-600 text-white dark:bg-green-500',
+  Done: 'bg-purple-600 text-white dark:bg-purple-500',
+  Planned: 'bg-blue-600 text-white dark:bg-blue-500',
+  Paused: 'bg-red-600 text-white dark:bg-red-500',
+  Deferred: 'bg-blue-600 text-white dark:bg-blue-500',
+  Unknown: 'bg-gray-600 text-white dark:bg-gray-500',
+};
+
+const effortColors = {
+  Low: 'bg-green-600 text-white dark:bg-green-500',
+  Medium: 'bg-yellow-600 text-white dark:bg-yellow-500',
+  High: 'bg-red-600 text-white dark:bg-red-500',
+  'Low-medium': 'bg-green-600 text-white dark:bg-green-500',
+  Unknown: 'bg-gray-600 text-white dark:bg-gray-500',
+};
+
+const impactColors = {
+  Low: 'bg-red-600 text-white dark:bg-red-500',
+  Medium: 'bg-yellow-600 text-white dark:bg-yellow-500',
+  High: 'bg-green-600 text-white dark:bg-green-500',
+  'Medium-high': 'bg-yellow-600 text-white dark:bg-yellow-500',
+  'Low-medium': 'bg-red-600 text-white dark:bg-red-500',
+  Unknown: 'bg-gray-600 text-white dark:bg-gray-500',
+};
+
+export default function Roadmap() {
+  const newTitle = 'Roadmap';
+  const markdownFile = '_indexPage';
+  const date = new Date().getFullYear();
+
+  return (
+    <SectionContext.Provider value='docs'>
+      <Head>
+        <title>{newTitle}</title>
+      </Head>
+      <Headline1>
+        {newTitle} {date}
+      </Headline1>
+
```

Comment:
```suggestion
      <p>
        Our "Roadmap" section displays our key objectives for the long term. While this roadmap provides a detailed outlook for the near future, please note that it might be subject to change. In fact, we are currently [discussing](https://github.com/orgs/json-schema-org/discussions/813) the new priorities for the next 24-month cycle, which will lead to relevant changes. Please consider joining the discussion to become an active part of JSON Schema's future!
      </p>
```